ext/CreateParametersExt.jl

```diff
@@ -220,4 +225,154 @@ function OptimalityFarquharParameters(toml_dict; kwargs...)
     return OptimalityFarquharParameters{FT}(; params..., kwargs..., mechanism)
 end
 
+
+"""
+    EnergyHydrologyParameters(
+        ::Type{FT};
+        ν,
+        ν_ss_om,
+        ν_ss_quartz,
+        ν_ss_gravel,
+        hydrology_cm,
+        K_sat,
+        S_s,
+        θ_r,
+        kwargs...,)
+
+    EnergyHydrologyParameters(
+        toml_dict;
+        ν,
+        ν_ss_om,
+        ν_ss_quartz,
+        ν_ss_gravel,
+        hydrology_cm,
+        K_sat,
+        S_s,
+        θ_r,
+        kwargs...,)
+
+EnergyHydrologyParameters has two constructors: float-type and toml dict based.
+Additional parameters must be added manually: ν, ν_ss_om, ν_ss_quartz, ν_ss_gravel, hydrology_cm, K_sat, S_s, and θ_r
+All parameters can be manually overriden via keyword arguments.
+"""
+function EnergyHydrologyParameters(
+    ::Type{FT};
+    ν,
+    ν_ss_om,
+    ν_ss_quartz,
+    ν_ss_gravel,
+    hydrology_cm,
+    K_sat,
+    S_s,
+    θ_r,
+    kwargs...,
+) where {FT <: AbstractFloat}
+    return EnergyHydrologyParameters(
+        CP.create_toml_dict(FT);
+        ν,
+        ν_ss_om,
+        ν_ss_quartz,
+        ν_ss_gravel,
+        hydrology_cm,
+        K_sat,
+        S_s,
+        θ_r,
+        kwargs...,
+    )
+end
+
+function EnergyHydrologyParameters(
+    toml_dict::CP.AbstractTOMLDict;
+    ν::F,
+    ν_ss_om::F,
+    ν_ss_quartz::F,
+    ν_ss_gravel::F,
+    hydrology_cm::C,
+    K_sat::F,
+    S_s::F,
+    θ_r::F,
+    PAR_albedo = 0.2,
+    NIR_albedo = 0.4,
+    kwargs...,
+) where {F <: Union{<:AbstractFloat, ClimaCore.Fields.FieldVector}, C}
```

Comment:
is it correct that these will be either floats or fieldvectors? I'm not as familiar with the parameters, but I would have expected floats or fields

Reply:
I'm not sure, this is just copied over from the old EnergyHydrologyParameters [constructor](https://github.com/CliMA/ClimaLand.jl/blob/main/src/standalone/Soil/energy_hydrology.jl#L84).